zerologr.go

```diff
@@ -33,10 +32,11 @@ var (
 	NameFieldName = "logger"
 	// NameSeparator separates names for logr.WithName
 	NameSeparator = "/"
+	// VerbosityFieldName is the field key for logr.Info verbosity
```

Comment:
Document that "" means not to emit that value?

Reply:
@thockin Fixed